pkg/controller/elasticsearch/services/services.go

```diff
@@ -50,6 +50,7 @@ func NewTransportService(es esv1.Elasticsearch) *corev1.Service {
 	labels := label.NewLabels(nsn)
 	ports := []corev1.ServicePort{
 		{
+			Name:     "tls-transport",
```

Comment:
This may deserve a little comment?
```suggestion
			Name:     "tls-transport", // name the service with the protocol for Istio
```